src/web/index.html

```diff
@@ -15,7 +15,13 @@
     <script src="/js/codemirror.js"></script>
     <script src="/js/q.js"></script>
     <script src="/js/require.js"></script>
+    
     <script>
+      // TODO: test these are the same as what was in old server version
+      var LOG_URL = "{{ &LOG_URL }}";
+      var GIT_REV = "{{ &GIT_REV }}";
+      var GIT_BRANCH = "{{ &GIT_BRANCH }}";
```

Comment:
Huh I don't know `&` vs. the way things are templated below. What/why the difference (I'm sure there's a good reason)?

Reply:
By default, for security, Moustache escapes a lot of stuff. The ampersand says "I trust the input and understand this will get interpolated raw into the source code, and accept responsibility for that". Without the ampersand, the LOG_URL is the main problem because it encodes the "/"s. If we thought any of these inputs could reasonably contain a literal `"` or `</script>` we'd need to be more careful, but since they don't, this is the easiest thing to do.